install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1646,8 +1646,10 @@ hubble:
       - auth_required
       - policy_denied
 
-# -- Method to use for identity allocation (`crd` or `kvstore`).
+# -- Method to use for identity allocation (`crd`, `kvstore` or `double-write`).
 identityAllocationMode: "crd"
+# -- Whether to read from the KVStore when using the Double-Write identity allocation mode (default true)
```

Comment:
Rather than another parameter, what about modes `kvstore2crd` and `crd2kvstore`?

And yes, I know naming debates are tedious. I just try and avoid new Helm values where possible.

Reply:
Yes, that's a good idea (similar to what Marco suggested yesterday: https://github.com/cilium/cilium/pull/31920#discussion_r1574442085)
I will refactor the code to have just one value.